ui/packages/catalog-ui-search/src/main/webapp/component/session-timeout/session-timeout.view.js

```diff
@@ -34,8 +34,12 @@ module.exports = Marionette.LayoutView.extend({
     }
   },
   serializeData: function() {
+    var countdownSeconds = sessionTimeoutModel.getIdleSeconds()
```

Comment:
it may be best to use `let` in this case since `countdownSeconds` isn't globally accessed